python/ray/scripts/scripts.py

```diff
@@ -54,6 +54,12 @@ def cli():
                     "maximum number of clients."))
 @click.option("--object-manager-port", required=False, type=int,
               help="the port to use for starting the object manager")
+@click.option("--redis-shard-ports", required=False, type=str,
+              help="the port to use for the Redis shards other than the "
+                   "primary Redis shard")
+@click.option("--object-store-memory", required=False, type=int,
```

Comment:
If you intend to do this change as well, mention in PR?

Reply:
Oh, I see it now.

Reply:
fixed